src/Layouts/Partials/TransformPanel.vala

```diff
@@ -0,0 +1,85 @@
+/*
+* Copyright (c) 2018 Alecaddd (http://alecaddd.com)
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 2 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+* Authored by: Ana Gelez <ana@gelez.xyz>
+*/
+public class Akira.Layouts.Partials.TransformPanel : Gtk.Grid {
+  construct {
+    border_width = 12;
+    row_spacing = 6;
+    column_spacing = 6;
+		hexpand = true;
+
+    attach (group_title (_("Position")), 0, 0);
+    attach (new Akira.Widgets.LinkedInput (C_("The horizontal coordinate", "X")), 0, 1, 1);
```

Comment:
What is the `C_` used btw? :) TIL that this is a thing

Reply:
It is like `_` but accepts another argument that will be shown to translators, to give them a bit of context about what they are translating (I used it here, because it is not obvious what you have to do when asked to translate a single letter :grin:)